src/components/screens/GuideScreen/Hero.js

```diff
@@ -207,7 +230,12 @@ Hero.propTypes = {
   description: PropTypes.string,
   heroAnimationName: PropTypes.string,
   imagePath: PropTypes.string,
-  languages: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
+  languages: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string,
+      tutorial: PropTypes.string,
+    })
```

Comment:
Is this required? My hunch is that it is.

```suggestion
    }).isRequired,
```

Reply:
you're right it should.

Reply:
i'll update accordingly and make the prop required, as it should, this will always be required, sorry about that, it was a oversight from my part.

Reply:
No worries at all.